gulpfile.js/tests/configGenerator.test.js

```diff
@@ -77,8 +78,8 @@ test('configGenerator - config properties should contain correct values', functi
         'config "label" should equal "Test-example"'
       )
       t.equal(
-        firstScenario.url, 'http://example:7000/section-test-example.html',
-        'config "url" should equal "http://example:7000/section-test-example.html"'
+        firstScenario.url, 'http://example:7000' + path.sep + 'section-test-example.html',
```

Comment:
Ah. These are expected to be URLs so should always be forward slashes. In this case the tests are right the code is probably wrong 😞